client/src/pages/Signup/SignupParents/SignupParents.tsx

```diff
@@ -299,6 +301,10 @@ const SignupParents = () => {
             </div>
           </FormGroup>
 
+          <Link to="/termsandconditions" className={styles.terms}>
```

Comment:
When i look in the UI on the forms then the submit button is cut and i can't see it all. why?